csi/moac/volume.ts

```diff
@@ -236,22 +267,32 @@ export class Volume {
       }
     }
 
-    // check that replicas are shared in the way they should be
-    let nexusNode, replicaSet;
-    try {
-      [nexusNode, replicaSet] = await this._ensureReplicaShareProtocols();
-    } catch (err) {
-      log.warn(err.toString());
+    // Unpublish the nexus if it should not be published
```

Comment:
Are we missing the `publish` counterpart?

Reply:
it is at line 362 in the new file. It is done later because we need to check a couple of things before we publish the volume.

Reply:
dohh see it now, thanks